pkg/api/plugin.go

```diff
@@ -125,4 +125,7 @@ type GenevaActions interface {
 
 	// ForceUpdate forces rotates all vms in a cluster
 	ForceUpdate(ctx context.Context, cs *OpenShiftManagedCluster, deployer DeployFn) *PluginError
+
+	// GetPluginVersion fetches the RP plugin version
+	GetPluginVersion(ctx context.Context, template *plugin.Config) ([]byte, error)
```

Comment:
I got a basic consistency complaint here: `GetPluginVersion` handles `/admin/pluginVersion` however the values come from the `config.ClusterVersion` (and the `clusterVersion` value in the config file). I'd much rather have all these follow the same nomenclature (e.g. `/admin/clusterVersion`) to avoid confusion in the future.

Reply:
@thekad you're right.  A future PR is going to rename config.ClusterVersion to PluginVersion.